CalendarPicker/Day.js

```diff
@@ -13,6 +13,14 @@ import { isSameDay } from 'date-fns/isSameDay';
 import { isWithinInterval } from 'date-fns/isWithinInterval';
 import { startOfDay } from 'date-fns/startOfDay';
 
+const DayComponent = memo(({ style, year, month, date, renderDay }) => {
```

Comment:
`day` isn't passed in as a parameter to this function. That's needed after moving outside of the Day component.
Please test this in the sample app 
https://github.com/stephy/CalendarPicker?tab=readme-ov-file#sample-application